pkg/broker/ingress/handler.go

```diff
@@ -97,15 +99,17 @@ type Handler struct {
 	decouple DecoupleSink
 	logger   *zap.Logger
 	reporter *metrics.IngressReporter
+	authType authcheck.AuthType
 }
 
 // NewHandler creates a new ingress handler.
-func NewHandler(ctx context.Context, httpReceiver HttpMessageReceiver, decouple DecoupleSink, reporter *metrics.IngressReporter) *Handler {
+func NewHandler(ctx context.Context, httpReceiver HttpMessageReceiver, decouple DecoupleSink, reporter *metrics.IngressReporter, authType authcheck.AuthType) *Handler {
 	return &Handler{
 		httpReceiver: httpReceiver,
 		decouple:     decouple,
 		reporter:     reporter,
 		logger:       logging.FromContext(ctx),
+		authType:     authType,
```

Comment:
I lost track - how is `authType` used in the `Handler`?

Reply:
It should be passed into ingress's probe check. However, as our ingress liveness and readiness check has an issue #1973, in this PR, I only configure authType to be an env into ingress Pod, but didn't add the probe check for it. I will do it after unblock #1973